spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -356,12 +356,13 @@ spring.cloud.deployer.cloudfoundry.task.apiTimeout=360
 ```
 
 Note that you can set the following properties `spring.cloud.deployer.cloudfoundry.services`,
-`spring.cloud.deployer.cloudfoundry.memory`, and `spring.cloud.deployer.cloudfoundry.disk` as part of an individual
-deployment request prefixed by the `app.<name of application>`.  For example
+`spring.cloud.deployer.cloudfoundry.buildpack` or the Spring Cloud Deployer standard
```

Comment:
I think the property `spring.cloud.deployer.cloudfoundry.services` is mentioned here because it is being used in one of the examples beneath. But, I don't see `spring.cloud.deployer.cloudfoundry.buildpack` being used in the examples anywhere.